server/routes/payment.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const {
+  createPaymentMethod,
+  getPaymentMethods,
+} = require("../controllers/payment");
+const { authenticate } = require("../middleware/authenticate");
+
+const Router = express.Router();
+
+Router.post("/methods", authenticate(), createPaymentMethod);
```

Comment:
I think the route can be more verbose, like payment-methods to avoid any confusion